### DISCUSSION TITLE
verticle tab accent color

### DISCUSSION THREAD

**Discussion by F4bC0d3** [Godiesc/firefox-one#18] [General]
verticle active tab accent color change without changing theme, what should in change in the TabCenterReborn-insideConfig.css

**Comment by Godiesc** [comment]
hi, that color is taken from the theme, don't change randomly. changes too when is a container tab.
You can checked in this lines: [code lines](https://github.com/Godiesc/firefox-one/blob/974fee10ce0ebc9b4025b90bb18d05d74c46230f/TabCenterReborn-insideConfig.css#L266C1-L278C2)

**Reply by F4bC0d3** [reply]
Yeah i understand that, i will keep a backup of anything goes bad, exactly which lines to change??? And how should i change??

**Reply by Godiesc** [reply]
![imagen](https://github.com/Godiesc/firefox-one/assets/22057609/f9d80180-ff86-44f0-b9c0-430e0a95ab4b)

**Reply by F4bC0d3** [reply]
Doesn't or maybe i am not doing it right, i want to change active tab colour not the pinned tab one. Can give a little brief explanation would be thankful.

**Reply by Godiesc** [reply]
https://github.com/Godiesc/firefox-one/assets/22057609/0476b1ef-457f-4d6f-a166-060dbd6b600c

**Reply by F4bC0d3** [reply]
Thnxx alot dude 🔥🔥
